### PR TITLE
Update label sync image and usage

### PR DESCRIPTION
**Pull request by spiffxp**
Based on a commit in the repo, with the usage that corresponds

### CODE REVIEW

File: label_sync/cluster/label_sync_job.yaml [hunk 1]
```diff
@@ -26,10 +26,10 @@ spec:
       restartPolicy: Never  # https://github.com/kubernetes/kubernetes/issues/54870
       containers:
       - name: label-sync
-        image: gcr.io/k8s-testimages/label_sync:v20171215-477dc7ebc
+        image: gcr.io/k8s-testimages/label_sync:v20180227-377685119
         args:
         - --config=/etc/config/labels.yaml
-        - --dry-run=false
```
Comment (ixdy) [inline label_sync/cluster/label_sync_job.yaml]: =true?

File: label_sync/cluster/label_sync_job.yaml [hunk 2]
```diff
@@ -26,10 +26,10 @@ spec:
...
+        - --confirm=false
```
Comment (ixdy) [inline label_sync/cluster/label_sync_job.yaml]: =true?

Comment (spiffxp) [reply label_sync/cluster/label_sync_job.yaml]: whoops thought I had pushed that, guess it was still local

**Pull request merged**